README.md

```diff
@@ -31,7 +31,7 @@ For automatic linking in templates you could create the templates:
 ## Database
 
 ```bash
-PGPASSWORD=secret psql -h localhost -U postgres -p 5432 -U zabbix << EOF
+PGPASSWORD=secret psql -h localhost -U postgres -p 5433 -U zabbix << EOF
```

Comment:
This is probably a typo from a dev environment?

Reply:
Yes, not sure how I managed to commit that.